libnopegl/src/node_rtt.c

```diff
@@ -201,6 +189,14 @@ static int get_renderpass_info(const struct ngl_node *node, int state, struct re
             if (state == RENDER_PASS_STATE_STOPPED)
                 info->nb_interruptions++;
             state = RENDER_PASS_STATE_STARTED;
+        } else if (child->cls->id == NGL_NODE_GRAPHICCONFIG) {
+            struct graphics_state graphics_state = {0};
+            ngli_node_graphicconfig_get_state(child, &graphics_state);
+            if (graphics_state.depth_test)
+                info->require_depth_buffer = 1;
+            if (graphics_state.stencil_test)
+                info->require_stencil_buffer = 1;
```

Comment:
I'd prefer to keep a features mask for requirements instead of bools

Reply:
Done.